sdk/formrecognizer/azure-ai-formrecognizer/CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 ## 3.2.0b3 (Unreleased)
 
 ### Features Added
+- Added samples showing how to use the `to_dict()` and `from_dict()` methods that can be found on the convenience models in v3.1 and v3.2-beta. Look for `sample_convert_to_and_from_dict.py` and `sample_convert_to_and_from_dict_async.py` in the samples directory under the `v3.1` and `v3.2-beta` directories depending on the version of the SDK library that is being used.
```

Comment:
We don't typically include new samples under "Features Added". It's been awhile since I've checked the guidelines on this, though

Reply:
I was unsure as well...but I do think we have a few instances of declaring new samples in the changelog. I'll look up the guidelines to see if there's any guidance

Reply:
Had an offline discussion about this...I'll remove this entry into the changelog.